internal/partnerplugin/partner_plugin.go

```diff
@@ -29,7 +29,8 @@ var _ PartnerPlugin = (*partnerPlugin)(nil)
 // Handles all communication with the partner plugin
 type PartnerPlugin interface {
 	DoServiceRequest(ctx context.Context, requestMsg *types.Message, service rpc.Client) (context.Context, *types.Message, error)
-	SendTokenBoughtNotification(ctx context.Context, tokenID *big.Int, mintID string, buyTxID common.Hash) error
+	SendTokenBoughtNotificationWithoutByTx(ctx context.Context, tokenID *big.Int, mintID string) error
```

Comment:
I guess that should be `[...]WithoutBuyTx`?

Reply:
ty, seems its lost in rebasing